README.md

```diff
@@ -8,3 +8,37 @@ This repository contains the source code for Zulip's PyPI packages:
   for Zulip's bots and bots API.
 * `zulip_botserver`: [PyPI package](https://pypi.python.org/pypi/zulip-botserver)
   for Zulip's Flask bot server.
+
+## Development
+
+1. Fork and clone the Git repo:
+   `git clone https://github.com/<your_username>/python-zulip-api.git`
+
+2. Make sure you have [pip](https://pip.pypa.io/en/stable/installing/)
+   and [virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
+   installed.
+
+3. `cd` into the repository cloned earlier:
+   `cd python-zulip-api`
+
+4. Run:
+   ```
+   ./tools/provision
+   source .zulip-api-env/bin/activate
```

Comment:
Yes. I'll push them soon. I'm still waiting for confirmation on whether the cygwin fix works as expected or not.